Jenkinsfile

```diff
@@ -163,9 +163,9 @@ pipeline {
                             script {
                                 // If this build is not a pull request, run owasp scan on the distribution. Otherwise run incremental scan
                                 if (env.CHANGE_ID == null) {
-                                    sh 'mvn verify -q -B -Powasp-dist -DskipTests=true -DskipStatic=true -pl !$DOCS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                                    sh 'mvn org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@directories dependency-check:check -q -B -Powasp-dist -DskipTests=true -DskipStatic=true -pl !$DOCS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```

Comment:
Could use feedback as to whether `dependency-check:aggregate` should be included here

Reply:
I think you would want to. 
https://jeremylong.github.io/DependencyCheck/dependency-check-maven/configuration.html
According to the documentation Aggregate makes a single report for the project and all it's children.

Reply:
👍 Okay I added it in to the non-PR build line